config/webpacker.yml

```diff
@@ -82,6 +82,7 @@ test:
 
 production:
   <<: *default
+  compress: false
```

Comment:
is the plan to turn this back on once assets are working in production?

Reply:
This was an experiment for the [assets not working issue](https://www.pivotaltracker.com/story/show/179566412) but did not have the desired (any?) effect. I can remove this.

Reply:
Responses..
- Created a chore for [restore Sentry source map uploads](https://www.pivotaltracker.com/story/show/180495753) and will do later (for the Azure version)
- Removed the `compress: false`
- No, will keep this repo on CI pipeline of Semaphore/Heroku
- N/A
- N/A